README.md

```diff
@@ -1,3 +1,107 @@
-# LearnGit
-Just Learning Git
+# CommandLineParser
+# Introduction
+[1].CommandLineParser library adds the user defined command line options which accepts the short command , long command , short description ,long description and a  default value.
 
+[2].This library parses the command line options and gives the corresponding value of a particular command.
+
+[3].The CommandLine parser Library supports custom help functionlity .
+
+[4].The command line Options can be of the following forms :- 
+
+    --port=3132
+    --port 3132
+    -p
+    -cp
+    -p 4215
+    
+[5].Example 
+
+let say the, resgistred command is port . 
+ 
```

Comment:
Avoid using sentences like "let say the ..". In technical documentation we generally we address third person.
Here we can write,
Port registration command
obj.............